Formula/seexpr.rb

```diff
@@ -22,47 +22,60 @@ class Seexpr < Formula
   depends_on "doxygen" => :build
   depends_on "libpng"
 
+  uses_from_macos "flex" => :build
+
+  on_linux do
+    depends_on "mesa"
+    depends_on "mesa-glu"
+  end
+
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DUSE_PYTHON=FALSE"
-      system "make", "doc"
-      system "make", "install"
-    end
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DUSE_PYTHON=FALSE
+      -DENABLE_LLVM_BACKEND=FALSE
+      -DENABLE_QT5=FALSE
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args
```

Comment:
```suggestion
    args = %W[
      -DCMAKE_INSTALL_RPATH=#{rpath}
      -DUSE_PYTHON=FALSE
      -DENABLE_LLVM_BACKEND=FALSE
      -DENABLE_QT5=FALSE
    ]

    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```
Pass `std_cmake_args` last to make sure you're not accidentally overriding it.